tutorials/llm/llama-3/pruning-distillation/README.rst

```diff
@@ -63,17 +73,38 @@ For pruning and distilling the model, you will use the NeMo Framework which is a
 
    jupyter lab --ip 0.0.0.0 --port=8888 --allow-root
 
-4. Then, navigate to `this notebook <./llama3-pruning-distillation-nemofw.ipynb>`_.
+4. Then, navigate to `this notebook <./introduction.ipynb>`_ to get started.
 
+This directory contains a list of notebooks which will go over all the steps to create a distilled 4B model.
```

Comment:
revise text

This directory contains a list of notebooks that cover all the steps to create a distilled 4B model.